src/routes/album/album.component.tsx

```diff
@@ -162,19 +186,49 @@ const Album = ({
                     <span className='text-sm mr-4 w-20'>Tags:</span>
 
                     <div className='flex items-center flex-wrap max-w-tags'>
-                      {album.tags.map(el => (
-                        <Link
-                          href={`/?page=1&tags=${el.id}`}
-                          passHref
-                          key={el.id}
+                      {!showMoreTags
```

Comment:
Create a new component and move tags logic there. Now it takes a lot of place and code becomes harder to read.